tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -50,6 +50,7 @@ const (
 //	at all times.
 const (
 	vmiPhaseCount                 = `sum by (phase) (kubevirt_vmi_phase_count{})`
+	vmiCountOnNode                = `sum(kubevirt_vmi_phase_count{node != "<no value>", node != ""}) by (node)`
```

Comment:
should this be using `sum_over_time({}[%ds])` instead of `sum` to capture the numbers of the current run?

I wonder if using just the `sum` metric will give us numbers from previous runs as well